release-notes.md

```diff
@@ -1,6 +1,28 @@
 # Release Notes
 
-## 3.1
+## 3.2.0
+
+### Detection for configuration overwrites
+
+In previous versions medic-conf would upload the new configuration overwriting whatever was there previously and there was no way to tell if you were accidentally wiping out someone elses changes. With this feature you will now be warned if another device has updated the configuration since you started making changes.
```

Comment:
Consider:

> This new feature warns and prompts users when uploading a configuration if the configuration they are overwriting has changed since their last upload. The warnings are prompts are comparable to a database write conflict.  Previous versions of medic-conf would upload the new configuration overwriting whatever was there, resulting in users accidentally wiping out someone elses changes or changes made through the admin console.

Reply:
Done.